unit-tests/py/rspy/libci.py

```diff
@@ -229,6 +230,10 @@ def name( self ):
     def ran( self ):
         return self._ran
 
+    @property
+    def context( self ):
```

Comment:
Hmm... why do we need to store the context? It's pretty much never different from test to test... do you foresee a need?

Reply:
Since we need the flag to run the test we either store it somewhere or we send it to the function that runs each test. I thought storing it was better.
I could maybe add a global variable to the libci module and use that instead. would that be better?